dashboard/src/components/model/InverterParameters.vue

```diff
@@ -111,25 +108,19 @@ export default class InverterParametersView extends ModelBase {
   @Prop() parameters!: SandiaInverterParameters | PVWattsInverterParameters;
   @Prop() model!: string;
   @Prop({ default: null }) selectedInverter!: string;
+  showBrowser = false;
 
   data() {
     return {
-      parameterSource: "User Supplied"
+      parameterSource: "User Supplied",
+      showBrowser: this.showBrowser
     };
   }
 
-  loadInverter(event: any) {
-    // TODO: load inverter from correct source based on selected model.
-    console.log(event.target.value);
-  }
-
-  get parameterOptions() {
-    // Return a list of parameter options based on currently selected model
-    if (this.model == "pvsyst") {
-      return ["PVSystInverter_0", "PVSystInverter_1"];
-    } else {
-      return ["PVWattsInverter_0", "PVWattsInverter_1"];
-    }
+  loadInverter(parameters: Record<string, any>) {
+    this.showBrowser = false;
+    // only supported for pvsyst model
+    this.parameters = new SandiaInverterParameters(parameters);
```

Comment:
this must not propagate correctly or something. If you add the parameters, then click Add Array, they disappear. If you add an array, then add parameters, then save, the parameters are not saved

Reply:
I think that failing test gets at this

Reply:
Thanks, this wasn't working correctly. Setting the whole Prop at once in inverter parameters was breaking the reference to the passed in `inverter_parameters`, so I've moved the event handling up a level, so the Inverter component can assign it's own parameters and then pass them back down for UI update. This is one of the tricky direct prop mutation issues. Okay to adjust the values that each object's properties point to but need to be careful to not destroy pointers.